client/verta/tests/unit_tests/deployment/test_build.py

```diff
@@ -92,3 +92,27 @@ def test_model_version_list_builds(
         sorted(build_dicts, key=lambda d: d["id"]),
     ):
         assert_build_fields(build, build_dict)
+
+
+@given(build_dict=build_dict())
+def test_build_start_scan_not_external_exception(mock_conn, build_dict):
+    """Test that start_scan() raises the expected error when the `external` parameter is not
+    included or False.  This can be removed when we support internal scans.
+    """
+    build = Build(mock_conn, build_dict)
+    with pytest.raises(
+        NotImplementedError,
+        match=(
+            "internal scans are not yet supported.  Please use `external=True`"
+            " parameter"
+        ),
+    ):
+        build.start_scan()
+    with pytest.raises(
+            NotImplementedError,
+            match=(
+                    "internal scans are not yet supported.  Please use `external=True`"
+                    " parameter"
+            ),
+    ):
```

Comment:
```suggestion
    with pytest.raises(
        NotImplementedError,
        match=(
            "internal scans are not yet supported.  Please use `external=True`"
            " parameter"
        ),
    ):
```

Reply:
Forgot to re-run black after the fix-up.  ~facepalm~

Reply:
Hmmmmm black does have other options for working with PyCharm 🤔 I wonder if one of them would work https://black.readthedocs.io/en/stable/integrations/editors.html